specification/marketplacecatalog/data-plane/Products/preview/2022-09-01-preview/products.json

```diff
@@ -0,0 +1,931 @@
+{
+  "x-generator": "NSwag v13.12.1.0 (NJsonSchema v10.4.6.0 (Newtonsoft.Json v13.0.0.0))",
+  "swagger": "2.0",
+  "info": {
+    "title": "Microsoft Marketplace Catalog Browse API",
+    "description": "Microsoft Marketplace Catalog Browse API",
+    "termsOfService": "None",
+    "version": "2022-09-01-preview"
+  },
+  "host": "catalogapi.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "paths": {
+    "/products/{id}": {
+      "get": {
+        "tags": [
+          "SinglePublicProduct"
+        ],
+        "summary": "Returns a single public product",
+        "operationId": "SinglePublicProduct_Get",
+        "x-ms-examples": {
+          "Product_Get": {
+            "$ref": "./examples/SinglePublicProduct_Get.json"
+          }
+        },
+        "parameters": [
+          {
+            "type": "string",
+            "name": "id",
+            "in": "path",
+            "required": true,
+            "description": "The unique product Id",
+            "x-nullable": true
```

Comment:
Please remove `x-nullable: true` here and everywhere else it appears in the file. This annotation should only be used if "null" has a special meaning that is different from the parameter/property not being set. And in this case the special meaning of "null" should be clearly described.